.circleci/config.yml

```diff
@@ -28,38 +28,28 @@ jobs:
            - v1-dependencies-
 
       - run: cd .clj-kondo && cat config.edn
+      - run: cat .cljfmt.edn
 
-      - run:
-          name: frontend styles prettier
-          working_directory: "./frontend"
-          command: |
-            yarn install
-            yarn run lint:scss
+      # - run:
```

Comment:
why is this commented?

Reply:
Because we do not want CI failing until we format all the code. And we can't  format all the code on this PR, because it probably will cause too many conflicts with ongoing work.